.changeset/brave-timers-lick.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fepack/image": minor
+"@fepack/react-image": minor
+---
+
+feat(image): add load to return image
```

Comment:
feat(image): add `LoadClient`, update `load` can return image, export all public apis of @fepack/image in @fepack/react-image